scipy/stats/tests/test_continuous_basic.py

```diff
@@ -79,6 +80,22 @@
 # distslow are sorted by speed (very slow to slow)
 
 
+# These distributions fail the complex derivative test below.
+# Here 'fail' mean produce wrong results and/or raise exceptions, depending
+# on the implementation details of corresponding special functions.
```

Comment:
I think mentioning the github issue would be useful, to find the explanation and discussion faster